### ISSUE TITLE
Add a BCC field for e-mails

### ISSUE THREAD

**Issue by lrt-danielat** [Simplero/Roadmap#49]
A BCC field for the e-mails and broadcasts, maybe even a BCC that we can set-up globally for our account would be very helpful for us. This would make it very easy to merge the information about the e-mails a contact received from our Simplero account with other CRMs that we're using.

Thanks a lot!

**Comment by calvincorreli** [comment]
I'm not sure I understand. This way, whenever you send a broadcast to a thousand recipients, the BCC address would get a thousand near-identical emails?

**Comment by christophcemper** [comment]
Yes, this would be needed to feed all those thousands of emails to the matching CRM records in our CRM.

We want to have a consistent view of all communication with the client, even if it comes from automated systems like simplero.
